<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: #eff4ff;
}</string>
  </property>
  <widget class="QLabel" name="register_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_lineedit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
  color: #1d1d1d;
  background-color: #ecf0f1;
  border: 2px solid #2c3e50;
  border-radius: 8px;
  padding-left: 25px;
  font-family:&quot;Microsoft YaHei UI&quot;;
  text-align: left;
  padding-right: 50px;
  background-image: url(:/qss/icon/password.png);
  background-repeat:no-repeat;
  background-position: center left;
}


</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: #1d1d1d;
  background-color: #bdc3c7;
	border:none;
  padding: 3px;
  font-family:&quot;Microsoft YaHei UI&quot;;
	text-align: center;
}
QPushButton:hover, QPushButton:pressed , QPushButton:checked
{
  background-color:  #95a5a6;
	text-align: right;
	padding-right: 20px;

}
QPushButton:hover
{
  background-image: url(:/qss/icon/cross.png);
	background-repeat:no-repeat;
	background-position: center left;
}
QPushButton:pressed, QPushButton:checked
{
  background-image: url(:/qss/icon/cross.png);
	background-repeat:no-repeat;
	background-position: center left;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="check_lineedit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
  color: #1d1d1d;
  background-color: #ecf0f1;
  border: 2px solid #2c3e50;
  border-radius: 8px;
  padding-left: 25px;
  font-family:&quot;Microsoft YaHei UI&quot;;
  text-align: left;
  padding-right: 50px;
  background-image: url(:/qss/icon/password.png);
  background-repeat:no-repeat;
  background-position: center left;
}


</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Check the psassword</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_lineedit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
  color: #1d1d1d;
  background-color: #ecf0f1;
  border: 2px solid #2c3e50;
  border-radius: 8px;
  padding-left: 25px;
  font-family:&quot;Microsoft YaHei UI&quot;;
  text-align: left;
  padding-right: 50px;
  background-image: url(:/qss/icon/account.png);
  background-repeat:no-repeat;
  background-position: center left;
}


</string>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QComboBox" name="user_type_combobox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
  color: #1d1d1d;
  background-color: #ecf0f1;
	border:2px solid #1d1d1d;
  padding: 3px;
  font-family:&quot;Microsoft YaHei UI&quot;;
	text-align: center;
}


  QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
 
    border-left-width: 1px;
    border-left-color: #1d1d1d;
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(:/qss/icon/double_down.png);
}


</string>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Cutomer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hotel Manger</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Platform Administrator</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: #1d1d1d;
  background-color: #bdc3c7;
	border:none;
  padding: 3px;
  font-family:&quot;Microsoft YaHei UI&quot;;
	text-align: center;
}
QPushButton:hover, QPushButton:pressed , QPushButton:checked
{
  background-color:  #95a5a6;
	text-align: right;
	padding-right: 20px;

}
QPushButton:hover
{
  background-image: url(:/qss/icon/register.png);
	background-repeat:no-repeat;
	background-position: center left;
}
QPushButton:pressed, QPushButton:checked
{
  background-image: url(:/qss/icon/register.png);
	background-repeat:no-repeat;
	background-position: center left;
}</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="belong_lineedit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
  color: #1d1d1d;
  background-color: #ecf0f1;
  border: 2px solid #2c3e50;
  font-family:&quot;Microsoft YaHei UI&quot;;
}


</string>
   </property>
  </widget>
  <widget class="QLabel" name="not_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
  color: #1d1d1d;
  background-color: none;
  border: none;
  font-family:&quot;Microsoft YaHei UI&quot;;

}</string>
   </property>
   <property name="text">
    <string>Not register your hotel?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: #1d1d1d;
  background-color: #bdc3c7;
	border:none;
  padding: 3px;
  font-family:&quot;Microsoft YaHei UI&quot;;
	text-align: center;
}
QPushButton:hover, QPushButton:pressed , QPushButton:checked
{
  background-color:  #95a5a6;
}</string>
   </property>
   <property name="text">
    <string>apply now</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="user_type_label">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
  color: #1d1d1d;
  background-color: none;
  border: none;
  font-family:&quot;Microsoft YaHei UI&quot;;

}</string>
   </property>
   <property name="text">
    <string>user type</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
  color: #1d1d1d;
  background-color: none;
  border: none;
  padding-left: 30px;
  font-family:&quot;Microsoft YaHei UI&quot;;
  text-align: left;
  background-image: url(:/qss/icon/error.png);
  background-repeat:no-repeat;
  background-position: center left;
}</string>
   </property>
   <property name="text">
    <string>error</string>
   </property>
  </widget>
  <zorder>register_label</zorder>
  <zorder>cancel_button</zorder>
  <zorder>not_label</zorder>
  <zorder>apply_button</zorder>
  <zorder>user_type_label</zorder>
  <zorder>error_label</zorder>
  <zorder>register_button</zorder>
  <zorder>belong_lineedit</zorder>
  <zorder>user_type_combobox</zorder>
  <zorder>check_lineedit</zorder>
  <zorder>password_lineedit</zorder>
  <zorder>username_lineedit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
